rasa/shared/importers/importer.py

```diff
@@ -100,8 +100,7 @@ def load_core_importer_from_config(
         importer = TrainingDataImporter.load_from_config(
             config_path, domain_path, training_data_paths, TrainingType.CORE
         )
-
-        return CoreDataImporter(importer)
+        return importer
```

Comment:
This allows us to check for e2e data in the NLU training data when using the `rasa core` command.

Reply:
The `get_stories` in `CoreDataImporter` just calls the `get_stories` of the importer it wraps (the one we are returning directly here). The only real difference with `CoreDataImporter` is that `get_nlu_data` is overridden to return an empty `TrainingData`.